app/controllers/games_controller.rb

```diff
@@ -61,16 +63,27 @@ def destroy
 
   private
 
+  def invalid_record(redirect_path:, resource:, message:)
+    respond_to do |format|
+      format.html { redirect_to redirect_path, alert: message }
+      format.json { render json: { errors: resource.errors }, status: :unprocessable_entity }
+    end
+  end
```

Comment:
Didn't we move that to the application controller? In which case we can just reuse the method.

Reply:
I overrode it to add a parameter: message. I read a bit more and realized that the error messages can be accessed via the resource that was already being passed in the original applicationController method. Removed this.

Reply:
future refactoring would be to make the error messages look a bit cleaner. Right now, here is what it looks like on the create game form:

![Screen Shot 2023-03-13 at 10 53 18 AM](https://user-images.githubusercontent.com/52888362/224738736-cb292153-3fe0-4834-a253-5e01b52ff14d.png)

I am happy that all errors are correctly displayed, but there is some serialization/parsing work to be done here. Perhaps a `ErrorSerializer` class that takes JSON and outputs messages based on values we store in the local file. Wdyt?

Reply:
responded on the new code